datafusion/physical-expr/src/equivalence/properties.rs

```diff
@@ -173,6 +174,12 @@ impl EquivalenceProperties {
         self.oeq_class.clear();
     }
 
+    /// Removes constant expressions that may change across partitions.
+    /// This method should be used when different partitions are fused.
```

Comment:
What does a "fused" partition mean? That is not a term I have run into previously

Reply:
What I mean is data from multiple partitions ends up in another partition (operators that does this `CoalescePartitionsExec`, `SortPreservingMergeExec`, `InterleaveExec`, `RepartitionExec`). I changed to term to "merge" in 747b69bf. However, If you have other suggestions which communicates the intent in this context better (maybe more common vocabulary in literature). I can update with that term.

Reply:
merge sounds good -- thank you